.github/CODEOWNERS

```diff
@@ -0,0 +1,35 @@
+# Automatically request reviews from maintainers
+# The CODEOWNERS are listed alphabetically
+
+# Package
+/pybamm/discretisations/ @martinjrobins @tinosulzer @rtimms
+/pybamm/experiment/ @brosaplanella @martinjrobins @tinosulzer @TomTranter @rtimms
+/pybamm/expression_tree/ @martinjrobins @tinosulzer @rtimms
+/pybamm/geometry/ @martinjrobins @tinosulzer @rtimms
+/pybamm/input/ @brosaplanella @DrSOKane @tinosulzer @TomTranter @rtimms
+/pybamm/meshes/ @martinjrobins @tinosulzer @rtimms
+/pybamm/models/ @brosaplanella @DrSOKane @tinosulzer @TomTranter @rtimms
+/pybamm/parameters/ @brosaplanella @DrSOKane @tinosulzer @TomTranter @rtimms
+/pybamm/plotting/ @martinjrobins @Saransh-cpp @tinosulzer @rtimms
+/pybamm/solvers/ @martinjrobins @tinosulzer @TomTranter @rtimms
+/pybamm/spatial_methods/ @martinjrobins @tinosulzer @rtimms
+/pybamm/step/ @brosaplanella @martinjrobins @tinosulzer @TomTranter @rtimms
+/pybamm/* @pybamm-team/maintainers # the files directly under /pybamm/
```

Comment:
Since rules lower down the list override ones above, wouldn't all maintainers get assigned for all the folders above? Maybe the /pybamm/* should go up higher?

Reply:
`/pybamm/*` will not recursively check every subdirectory, but will only check the files directly under `pybamm/` -

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```

Reply:
Works for me then